recipes/itk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class HelloTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    name = "example"
+    version = "4.13.2"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def requirements(self):
+        self.requires("itk/{0}".format(self.version))
+
+    def imports(self):
```

Comment:
```suggestion
```

Reply:
Is there any reason for keeping imports? If not, remove it. The `run_environment=True` is able to solve all dynamic libs paths.